packages/slate-editor/src/modules/nodes-hierarchy/fixers/insertTableRow.ts

```diff
@@ -1,23 +1,17 @@
+import { TablesEditor } from '@prezly/slate-tables';
 import { Transforms } from 'slate';
 import type { NodeEntry } from 'slate';
 import type { Editor } from 'slate';
 
-import { createParagraph } from '#extensions/paragraphs';
-import { createTableCellNode, createTableRowNode } from '#extensions/tables';
-
 export function insertTableRow(editor: Editor, [node, path]: NodeEntry) {
-    Transforms.insertNodes(
-        editor,
-        [
-            createTableRowNode({
-                children: [createTableCellNode({ children: [createParagraph()] })],
-            }),
-        ],
-        {
+    if (TablesEditor.isTablesEditor(editor)) {
+        Transforms.insertNodes(editor, [TablesEditor.createTableRow(editor, { children: 3 })], {
```

Comment:
Should we maybe just remove the table without any rows?

This will be consistent with what we do with empty rows in `slate-tables`: `removeEmptyRows()`.